h/services/group.py

```diff
@@ -192,9 +192,8 @@ def _create(self, name, userid, description, type_flags,
         """
         creator = self.user_fetcher(userid)
         scopes = [GroupScope(origin=o) for o in origins]
-        if organization is None:
-            organization = Organization.default(self.session)
-        self._validate_authorities_match(creator.authority, organization.authority)
+        if organization is not None:
```

Comment:
The group _service_ no longer imposes an organization by default, but the schema and view handler for the admin create-open-or-restricted-group form still requires it, so that should, I hope, keep people out of trouble.